constraints.txt

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
I think I'd prefer this in a separate `requirements/` dir.

Reply:
I would like to hide many of the files from root under a subfolder but picking `requirements/` would not be a generic enough approach. Lets delay this until we can make a more generic solution, maybe under `.config/`?